### PR TITLE
docs: adding tip on configuration for healthcheck

### PR DESCRIPTION
**Pull request by EvertonSA**
Goal:
I as a airflow deployer would like to enable kubernetes liveness probes. 

Issue:
On my installation, worker_enable_remote_control was set to False, leading to following return:

```
airflow@airflow-worker-0:/opt/airflow$ celery -A airflow.providers.celery.executors.celery_executor.app inspect ping
/home/airflow/.local/lib/python3.11/site-packages/airflow/metrics/statsd_logger.py:184 RemovedInAirflow3Warning: The basic metric validator will be deprecated in the futu
re in favor of pattern-matching.  You can try this now by setting config option metrics_use_pattern_match to True.
Error: No nodes replied within time constraint
```

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: docs/apache-airflow/administration-and-deployment/logging-monitoring/check-health.rst [hunk 1]
```diff
@@ -143,6 +143,11 @@ CLI Check for Celery Workers
 To verify that the Celery workers are working correctly, you can use the ``celery inspect ping`` command. On failure, the command will exit
 with a non-zero error code.
 
+.. note::
+
+  For this check to work, ``[celery]worker_enable_remote_control`` must be ``True``. 
+  If the parameter is set to ``False`` the command will exit with a non-zero error code.
```
Comment (uranusjr) [inline docs/apache-airflow/administration-and-deployment/logging-monitoring/check-health.rst]:
```suggestion
  If the parameter is set to ``False``, the command will exit with a non-zero error code. 
```

File: docs/apache-airflow/administration-and-deployment/logging-monitoring/check-health.rst [hunk 2]
```diff
@@ -156,3 +161,4 @@ To check if the all workers in the cluster running is working correctly, run:
     celery --app airflow.providers.celery.executors.celery_executor.app inspect ping
 
 For more information, see: `Management Command-line Utilities (inspect/control) <https://docs.celeryproject.org/en/stable/userguide/monitoring.html#monitoring-control>`__ and `Workers Guide <https://docs.celeryproject.org/en/stable/userguide/workers.html>`__ in the Celery documentation.
+
```
Comment (uranusjr) [inline docs/apache-airflow/administration-and-deployment/logging-monitoring/check-health.rst]:
```suggestion
```

**Review by uranusjr — APPROVED** [review]

**Comment by EvertonSA** [comment]
i can fix later thanks for reviewing!

File: docs/apache-airflow/administration-and-deployment/logging-monitoring/check-health.rst [hunk 3]
```diff
@@ -143,6 +143,11 @@ CLI Check for Celery Workers
 To verify that the Celery workers are working correctly, you can use the ``celery inspect ping`` command. On failure, the command will exit
 with a non-zero error code.
 
+.. note::
+
+  For this check to work, ``[celery]worker_enable_remote_control`` must be ``True``. 
+  If the parameter is set to ``False``, the command will exit with a non-zero error code.
```
Comment (uranusjr) [inline docs/apache-airflow/administration-and-deployment/logging-monitoring/check-health.rst]:
```suggestion
.. note::

  For this check to work, ``[celery]worker_enable_remote_control`` must be ``True``.
  If the parameter is set to ``False``, the command will exit with a non-zero error code.
```

**Comment by EvertonSA** [comment]
@uranusjr thank you for the fixes!

**Review by eladkal — APPROVED** [review]

**Pull request merged**